packages/auth/amplify_auth_cognito_dart/lib/src/model/signup/cognito_sign_up_result.dart

```diff
@@ -23,11 +23,9 @@ class CognitoSignUpResult extends SignUpResult {
   const CognitoSignUpResult({
     required super.nextStep,
     required super.isSignUpComplete,
-    this.userId,
+    super.userId,
   });
 
-  /// The user ID of the signed-up user.
-  ///
-  /// Will not be present for `Amplify.Auth.confirmSignUp` calls.
```

Comment:
Removed this comment - we'll need to add it to docs

Reply:
Its a pretty goofy pattern, but you could override the field to change the comment for this class. Not sure of any other way to do it. I'm fine either way.

<img width="459" alt="Screen Shot 2022-11-30 at 11 11 12 AM" src="https://user-images.githubusercontent.com/20613561/204849713-be584db7-15cc-4d63-b2ee-ca74f7fdfb00.png">